### PR TITLE
Added blurbs to config.json

### PR DESCRIPTION
**Pull request by yzAlvin**

### CODE REVIEW

File: concepts/chars/.meta/config.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
-  "blurb": "TODO: add blurb for chars concept",
+  "blurb": "A char is a type that respresents a Unicode UTF-16 character. Strings are a sequence of chars.",
```
Comment (ErikSchierboom) [inline concepts/chars/.meta/config.json:2]:
```suggestion
  "blurb": "A char is a type that represents a Unicode UTF-16 character. Strings are a sequence of chars.",
```

File: concepts/classes/.meta/config.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
 {
-  "blurb": "TODO: add blurb for classes concept",
+  "blurb": "A class in C# is made up of it's fields and methods. They are templates for creating objects with.",
```
Comment (ErikSchierboom) [inline concepts/classes/.meta/config.json:2]:
```suggestion
  "blurb": "A class in C# is made up of its fields and methods. They are templates for creating objects with.",
```

**Review by ErikSchierboom — COMMENTED** [review]
Some minor suggestions, but looks good!

**Comment by yzAlvin** [comment]
Oops, thanks for that

**Comment by yzAlvin** [comment]
Would it be helpful to keep track of a checklist of which concepts have blurbs?

- [x] arrays
- [x] attributes
- [ ] basics
- [ ] bit-manipulation
- [x] booleans

etc..

**Comment by ErikSchierboom** [comment]
@yzAlvin Yeah that could be really useful!

**Review by ErikSchierboom — APPROVED** [review]
I would prefer to merge this and then do follow-up PRs, okay?

**Comment by yzAlvin** [comment]
Yeah sure thing, for that checklist thing would I just copy+paste and make edits each new PR?

**Comment by ErikSchierboom** [comment]
@yzAlvin I've created the checklist: https://github.com/exercism/csharp/issues/1538 I don't think you'll have permissions, so I'll update it.

**Comment by iHiD** [comment]
@yzAlvin Awesome work. Thank you!!

**Pull request merged**